protocol/agent-protocol.md

```diff
@@ -261,27 +287,41 @@ error = %s"ERR" SP <errorType>
 
 #### NEW command and INV response
 
-`NEW` command is used to create a connection and an invitation to be sent out-of-band to another protocol user. It should be used by the client of the agent that initiates creating a duplex connection.
+`NEW` command is used to create a connection and an invitation to be sent out-of-band to another protocol user (the accepting party). It should be used by the client of the agent that initiates creating a duplex connection (the initiating party).
 
-`INV` response is sent by the agent to the client.
+`INV` response is sent by the agent to the client of the initiating party.
 
 #### JOIN command
 
-It is used to create a connection and accept the invitation received out-of-band. It should be used by the client of the agent that accepts the connection.
+It is used by to create a connection and accept the invitation received out-of-band. It should be used by the client of the agent that accepts the connection (the joining party).
+
+#### REQ notification and ACPT command
 
-#### CON notification
+When the accepting party uses `JOIN` command, the initiating party will receive `REQ` notification with some numeric identifier and an additional binary information, that can be used to identify the joining party or for any other purpose.
```

Comment:
align across commands - either "the accepting party", or "the joining party" - right now they randomly alternate creating unnecessary confusion